stellar_helpers.py

```diff
@@ -0,0 +1,71 @@
+from decimal import Decimal
+from stellar_sdk import TransactionBuilder, Server, Network, Keypair, Account, asset
+import os
+import sys
+from stellar_sdk.operation.create_claimable_balance import Claimant
+from stellar_sdk.time_bounds import TimeBounds
+from stellar_sdk.xdr.base import String
+
+STELLAR_USE_TESTNET = 'USE_STELLAR_TEST_NET' in os.environ
+STELLAR_ENDPOINT = "https://horizon-testnet.stellar.org" if STELLAR_USE_TESTNET else "https://horizon.stellar.org"
+STELLAR_PASSPHRASE = Network.TESTNET_NETWORK_PASSPHRASE  if STELLAR_USE_TESTNET else Network.PUBLIC_NETWORK_PASSPHRASE
+
+PUBLIC_KEY = os.environ['REWARD_PUBLIC_KEY']
+
+server = Server(horizon_url=STELLAR_ENDPOINT)
+
+def fetch_account_balance(pubKey = PUBLIC_KEY):
+    """
+    Returns the balance of the given account available to be send 
+    """
+    acc = server.accounts().account_id(PUBLIC_KEY).call()
+    balance = 0
+    
+    for b in acc['balances']:
+        if b['asset_type'] == 'native':
+            balance = float(b['balance'])
+            break
+    
+    balance -= 0.5 * (2 + acc['subentry_count']) # base reserve
+    print(f"{balance}")
+    return balance
+
+def generate_reward_tx(rewardee):
+    """
+    Generates transaction XDR full of claimable balance operations
+    takes tewardee (array of tuples (pub_key, amount)) where pub_key is str and amount is float
+    returns XDR as string
+    """
+    source_acc = Account(account_id=PUBLIC_KEY, sequence=1) # fetch sequence or will it be used with a fee bump?
+
+    tx = TransactionBuilder(
+        source_account=source_acc,
+        network_passphrase=STELLAR_PASSPHRASE,
+        base_fee=100)\
```

Comment:
fetch base fee here or will this be bumped anyway? (same with sq above)

Reply:
We should probably just fetch the base fee tbh. While it's not a huge trouble to wrap it in a fee bump, it's an extra step we can leave out.

Reply:
The way it is implemented atm a fee bump is required as the seq number is set to 1

Reply:
Oh didn't notice that. Not that a fee bump wouldn't help us in that case, as it just bumps the fee. The sequence number is still consumed from the source account of the inner tx

Reply:
In that case just assume the seq won't change inbetween issuing this tx and Tyler transmitting it?

Reply:
Well be using a dedicates account for this anyways. So I don't think the suquence number will change in between.
But we could again just ask Tyler 😉